lesson03/assignment/css/style.scss

```diff
@@ -4,3 +4,127 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// color library
+$base-gray: #5a5654;
+$shadow-gray: #bab4b1;
+
+// font styles
+$heading: bold;
+$occupation: lighter;
+
+// mobile responsiveness
+$breakpoint-mobile: "mobile";
+$breakpoint-ipad: "ipad";
+$breakpoint-small: "small";
+
+// border radius
+@mixin border-radius($radius: .8rem) {
+  -webkit-border-radius: $radius;
+  -moz-border-radius: $radius;
+    border-radius: $radius;
+}
+
+// box shadow for outside and inset
+@mixin box-shadow($horiz : .1em , $vert : .1em , $blur : .7em , $color : $shadow-gray , $inset : false) {
+  @if $inset {
+    -webkit-box-shadow:inset $horiz $vert $blur $color;
+    -moz-box-shadow:inset $horiz $vert $blur $color;
+    box-shadow:inset $horiz $vert $blur $color;
+  } @else {
+    -webkit-box-shadow: $horiz $vert $blur $color;
+    -moz-box-shadow: $horiz $vert $blur $color;
+    box-shadow: $horiz $vert $blur $color;
+  }
+}
+
+.main {
+  @media screen and (max-width: 20em) {
+    padding: 1rem;
+  }
+}
+
+// employee card main container
+.employee-card {
+  max-width: 1020px;
+  @include border-radius;
+  @include box-shadow;
+  display: flex;
+  border: solid 3.2rem lighten( $base-gray, 40%);
+  margin-bottom: 3rem;
+  padding: 2rem;
+
+  @include media-query($breakpoint-mobile) {
+    flex-wrap: wrap;
+    border-width: 1.5rem;
+    padding: 1rem;
+  }
+}
+
+// employee image, name and title container
+.name-container {
+  text-align: center;
+  img {
+    border-radius: 50%;
+    width: 225px;
```

Comment:
Remember that a core tenet of responsive design is relative units (%, rem), not static (px).